Bali/src/main/java/Offer.java

```diff
@@ -0,0 +1,121 @@
+import java.util.ArrayList;
```

Comment:
By convention, you are expected to put your Java files to a package. A package usually corresponds to the name of the group/artifact